dts/bindings/adc/ti,ads114s08.yaml

```diff
@@ -1,58 +1,8 @@
 # Copyright (c) 2023 SILA Embedded Solutions GmbH
 # SPDX-License-Identifier: Apache-2.0
 
-description: Texas Instrument 12 channels 16 bit SPI ADC
+description: Texas Instrument 12 channels 24 bit SPI ADC
```

Comment:
Shouldn't this be the 16 bit version?

Reply:
Done.

Reply:
done
Could you please close this conversation, as well as the others where we have agreed upon something? Unfortunately I cannot do this by myself, and having all these open conversations in the review is distracting from the actual open issues.